out/can_msg.ts

```diff
@@ -0,0 +1,47 @@
+
+// For setting the CAN device
+const enum CanDevice {
```

Comment:
Add the `export` keyword to these or they aren't accessible/visible to other modules via require.js